### ISSUE TITLE
Fix cicd pipeline telemetry issue

### ISSUE THREAD

**Issue by bricerisingalgorand** [algorand/go-algorand#2018]
## Summary
The cicd pipeline is showing thousands of backslashes in the logs and the job isn't reporting to datadog.

## Scope/Requirements
* Fix the job so that the logs in the job look right and perf metrics are reported to datadog

## Urgency/Relative Priority
This needs to be done now because it is blocking Nicholas in the perf10x work

**Comment by bricerisingalgorand** [comment]
I found this in the jenkins log `FORBIDDEN/12/index read-only / allow delete (api)` and noticed that the telemetry index is in read only mode. I checked the performance telemetry server and saw it was at 95% disk util. My guess is that elastic realized it is out of disk and switched the indexes to read-only in order to preserve the host. I deleted about 30% of the telemetry data and asked Nicholas to start a job to see if the issue is resolved.
